### PR TITLE
Python3 update

### PR DESCRIPTION
**Pull request by mcdonnnj**
This PR should allow cyhy-feeds to run in either a Python 2 or Python 3 environment. This is mostly just updates to print statements, but there were changes in configparser that required additional modifications.

### CODE REVIEW

File: aws_jobs/cyhy-data-extract.py [hunk 1]
```diff
@@ -22,7 +22,11 @@
 
 import re
 import sys
-from ConfigParser import SafeConfigParser
+
```
Comment (dav3r) [inline aws_jobs/cyhy-data-extract.py:27]: Please add a comment here explaining what is going on

File: aws_jobs/cyhy-data-extract.py [hunk 2]
```diff
@@ -22,7 +22,12 @@
...
```
Comment (jsf9k) [inline aws_jobs/cyhy-data-extract.py:27]: I'd probably move this after `import os` since `SafeConfigParser` is the thing being imported.

**Review by jsf9k — APPROVED** [review]
Excellent work!  I made one very minor suggestion that you can feel free to take or leave.

File: aws_jobs/cyhy-data-extract.py (hunk 2 above)
Comment (mcdonnnj) [reply aws_jobs/cyhy-data-extract.py:27]: Adjusted that and any similar imports per your feedback.

File: aws_jobs/cyhy-data-extract.py [hunk 3]
```diff
@@ -375,7 +378,7 @@ def main():
             ES_AWS_CONFIG_SECTION_NAME,
         )
     )
-    json_filename = "{!s}_{!s}.json".format(
+    json_filename = "{}_{!s}.json".format(
```
Comment (dav3r) [inline aws_jobs/cyhy-data-extract.py]: Is there any reason why shouldn't we just do it like this?
  `json_filename = "DMARC_{!s}.json".format(...`

**Review by dav3r — APPROVED** [review]
This all looks great, aside from two minor requests (see my comments). 👍

File: aws_jobs/cyhy-data-extract.py (hunk 1 above)
Comment (dav3r) [reply aws_jobs/cyhy-data-extract.py:27]: I see you already did this in the time between when I wrote my comment and when I finally submitted my review. 👍

File: aws_jobs/cyhy-data-extract.py (hunk 3 above)
Comment (mcdonnnj) [reply aws_jobs/cyhy-data-extract.py]: None that I can so I have changed it accordingly.

**Pull request merged**